synapse/storage/databases/main/devices.py

```diff
@@ -2050,49 +2071,25 @@ async def add_device_list_outbound_pokes(
         user_id: str,
         device_id: str,
         room_id: str,
-        stream_id: Optional[int],
         hosts: Collection[str],
         context: Optional[Dict[str, str]],
     ) -> None:
         """Queue the device update to be sent to the given set of hosts,
         calculated from the room ID.
-
-        Marks the associated row in `device_lists_changes_in_room` as handled,
-        if `stream_id` is provided.
         """
+        if not hosts:
+            return
 
         def add_device_list_outbound_pokes_txn(
             txn: LoggingTransaction, stream_ids: List[int]
         ) -> None:
-            if hosts:
```

Comment:
> Why are we removing these? Is these baths no longer used?
>
> *-- @erikjohnston, https://github.com/matrix-org/synapse/pull/14516/files#r1029502488*

What does "baths" mean here?

Reply:
I think that was meant to "Are these paths..."

Reply:
One day I will english good